### ISSUE TITLE
[Feature Request] More distribution methods

### ISSUE THREAD

**Issue by Arudarin** [ston3o/netclix#6]
It would be pretty nice to have distribution methods for the big players in OS's, e.g. a .msi or .exe for Windows, brew for MacOS, ppa for Debian based distros, rpm for Fedora based distros, and AUR package for Arch Linux based distros.

**Comment by johackim** [comment]
Yes, it could be cool, I tried to package this project with https://github.com/zeit/pkg but I have a problem to use it with phantomjs https://github.com/zeit/pkg/issues/147.

So for now it only available with npm and yaourt.

**Comment by Arudarin** [comment]
Can you change the yaourt in the README to pacaur or something that doesn't source the PKGBUILD's _before_ you read it? Serious security risk.

**Comment by johackim** [comment]
@ThatCynicalX Oh yes, sorry about your PR ! I closed it accidentally. You can open a new PR with pacaur replacement if you want :)

**Comment by Arudarin** [comment]
Alright!
